fieldpath/element_test.go

```diff
@@ -91,7 +91,7 @@ func TestPathElementLess(t *testing.T) {
 		}, {
 			name: "FieldName-3",
 			a:    PathElement{FieldName: strptr("capybara")},
-			b:    PathElement{Key: &value.Map{Items: []value.Field{{Name: "dog", Value: value.IntValue(3)}}}},
+			b:    PathElement{Key: &[]KeyValue{{"dog", 3}}},
```

Comment:
hm, it'd be nice to use the KeyByFields function. Maybe declare a local:

```
func kvlptr(args... interface{}) *KeyValueList {
  kvl := KeyByFields(args...)
  return &kvl
}
```

or something?